includes/movienight

```diff
@@ -10,21 +10,22 @@ name="movienight"
 rcvar="${name}_enable"
 command="/usr/local/${name}/MovieNight -f /usr/local/${name}/settings.json"
 daemon_user="movien"
-managerPid="/var/run/${name}.manager.pid
+managerPid="/var/run/${name}.manager.pid"
 mnPid="/var/run/${name}.pid"
 
 start_cmd="daemon_start"
 stop_cmd="daemon_stop"
 status_cmd="daemon_status"
 
 daemon_start() {
-	/usr/sbin/daemon -P ${managerPid} -p ${mnPid} -r -f -u $daemon_user $command
+	cd /usr/local/${name}
+	/usr/sbin/daemon -P ${managerPid} -p ${mnPid} -r -f -S -o /var/log/${name}.log -u ${daemon_user} ${command}
```

Comment:
I don't get the utility of the usage of the option `-o`, because the logging file of MovieNight can set into `setting.json` through the option `LogFile`.

Reply:
Add daemon error logging to /var/log if an error is generated before movienight runs

Reply:
Yes ! you're right, I forget about that.